lib/matplotlib/tests/test_axes.py

```diff
@@ -3595,6 +3592,14 @@ def test_move_offsetlabel():
     ax.yaxis.tick_right()
     assert_equal((1, 0.5), ax.yaxis.offsetText.get_position())
 
+@image_comparison(baseline_images=['square_plot'], extensions=['png'])
+def test_quiver_easy():
```

Comment:
Ah yes, will do, thank you for the input, that was the remnants of a previous test.